### ISSUE TITLE
false positive helpful-string

### ISSUE THREAD

**Issue by KotlinIsland** [KotlinIsland/basedmypy#768]
```py
def f[T, T2](t: T, t2: T2) -> T | T2:
    return t


print(f"{f("", "")}")  # The string for "object" isn't helpful in a user-facing or semantic string
```
